CMakeLists.txt

```diff
@@ -290,8 +290,6 @@ set(os_files
     ${nupic_src}/os/Directory.hpp
     ${nupic_src}/os/Env.cpp
     ${nupic_src}/os/Env.hpp
-    ${nupic_src}/os/FStream.cpp
-    ${nupic_src}/os/FStream.hpp
```

Comment:
Did you actually remove the files? I don't see it in the PR, but might be github just doesn't show it. 
Great work and nice commit (it'll be easy to merge).